webview-ui/src/components/prompts/PromptsView.tsx

```diff
@@ -1118,6 +1223,49 @@ const PromptsView = ({ onDone }: PromptsViewProps) => {
 									letters, numbers, and hyphens.
 								</div>
 							</div>
+							<div style={{ marginBottom: "16px" }}>
+								<div style={{ fontWeight: "bold", marginBottom: "4px" }}>Save Location</div>
+								<div
+									style={{
```

Comment:
Nit: Can we use tailwind instead of style objects for net-new markup?

In this case, something like:

```tsx
<div className="text-md text-vscode-descriptionForeground mb-2" />
```

Note that vscode CSS vars need to be added to index.css in order to be used in tailwind, and we only have a dozen or so currently (so you'd add `text-vscode-descriptionForeground` in this case).